gradio/queueing.py

```diff
@@ -376,6 +381,7 @@ def log_message(
         event_id: str,
         log: str,
         level: Literal["info", "warning"],
+        duration: int | None = 10000,
```

Comment:
why 10000 by default?

Reply:
Originally had it being set in ms and forgot to change the default. Fixed now.